index.html

```diff
@@ -269,7 +269,7 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
           </div>
         </div>
       </section>
-      <section class="contact-card-section">
+      <form class="contact-card-section">
```

Comment:
- [ ] Please make sure to follow the figma template, currently I can see some differences:

Figma  | Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/149946629-13f69b5b-3aba-43ae-982c-4f15e2a06dfa.png) | ![image](https://user-images.githubusercontent.com/77038610/149946585-7894287c-d834-48b4-b6ad-49b269150bb6.png)

- Please delete the white border around the form
- Please center the text of the `<h1>` and `<p>` elements
- Please align the placeholder text of the text input
- Please make sure to have similar margins and paddings